tests/test_cli.py

```diff
@@ -892,6 +892,15 @@ def test_release_url_metadata_python(self):
             'platform': None,
             'format': 'egg',
         })
+        url = 'https://files.pythonhosted.org/packages/26/01/0330e3ba13628827f10fcd6c3c8d778a5aa3e4d0a09d05619f074ba2d87e/nltk-3.2.4.win32.exe'
+        self.assertEqual(hashin.release_url_metadata(url), {
+            'package': 'nltk',
+            'version': '3.2.4',
+            'python_version': mock.ANY,
```

Comment:
The structure of `nltk-3.2.4.win32.exe` does not contain a python version so technically the `release_url_metadata` returns python_version=win32. I'm not going to change the regex any further because I don't know what the consequences of making the python_version optional are (there are not enough tests here to ensure this doesn't break anything).